src/core/radio/radio.hpp

```diff
@@ -537,6 +537,15 @@ class Radio : public InstanceLocator, private NonCopyable
      *
      */
     Error EnableCsl(uint32_t aCslPeriod, otShortAddress aShortAddr, const otExtAddress *aExtAddr);
+
+    /** Resets CSL receiver in radio.
+     *
```

Comment:
```suggestion
    /**
     * Resets CSL receiver in radio.
     *
```

Reply:
fixed for enable csl implementation as well